frontend/components/learning_circle_form/DayTimeSection.jsx

```diff
@@ -17,7 +19,7 @@ export default class  DayTimeSection extends React.Component {
   render(){
     const props = this.props;
     let reminderWarning = null;
-    let minDate = null;
+    let minDate = new Date();
```

Comment:
@s-kennedy is there a reason why you set the date here? This prevents creating learning circles in the past. Current functionality is to allow creation of learning circles in the past, but to disallow editing it into the past. The motivation was to allow capturing prior learning circles if needed, but stop surprising things from happening